client/src/Pages/Landing/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Container, Divider, Typography } from '@material-ui/core';
+
+import CardContainer from '../../Components/CardContainer';
+import useStyles from './style';
+
+function Landing() {
+  const classes = useStyles();
+
+  const [houses, setHouses] = useState([]);
+  const [errorMsg, setErrorMsg] = useState('');
+
+  const sortByTopRating = (housesData) => {
+    const cloningHouse = [...housesData];
+    return cloningHouse.sort((a, b) => {
+      if (a.rating > b.rating) {
+        return -1;
+      }
+      if (a.rating < b.rating) {
+        return 1;
+      }
+      return 0;
+    });
+  };
+
+  const sortByRecentlyAdded = (housesData) => {
+    const cloningHouse = [...housesData];
+    return cloningHouse.sort((a, b) => {
+      if (a.created_at > b.created_at) {
+        return -1;
+      }
+      if (a.created_at < b.created_at) {
+        return 1;
+      }
+      return 0;
+    });
+  };
+
```

Comment:
remove this function, duplicated logic , you already sort by id that gets that newest houses